tests/integration/conftest.py

```diff
@@ -167,8 +167,9 @@ async def test_charm_fixture(
 
     yield app
 
-    logger.info("Cleaning up test charm.")
-    await model.remove_application(app_name=app_name)
+    # logger.info("Cleaning up test charm.")
+    # await model.remove_application(app_name=app_name, force=True, no_wait=True)
+    # logger.info("Test charm removed.")
```

Comment:
In a follow up PR could we please remove this commented out code?

I'm not suprised this is flaky, this has never been reliable from memory

Reply:
Sure I'll replace the comments with something in the lines of - "due to the bug the cleanup must be done manually on OpenStack"